analysis/technical-analysis.md

```diff
@@ -0,0 +1,471 @@
+launching a requested process and may refuse to launch a process that is not authorized or authenticated.
+* **Programmability**: Some TEEs may be programmed with arbitrary code, while some may only support
+a limited set of operations. A TEE might even include or be composed entirely of code fixed at the time
+of manufacture.
+* **Attestability**: Often, a TEE can provide evidence or measurements of its origin and current state,
+so that the evidence can be verified by another party and—programmatically or manually—it can
+decide whether to trust code running in the TEE. It is typically important that such evidence is signed
+by hardware that can be vouched for by a manufacturer, so that the party checking the evidence
+has strong assurances that it was not generated by malware or other unauthorized parties. (Further
+discussion on attestation is provided later in section 7.)
+* **Recoverability**: Some TEEs may provide a mechanism for recovery from a non-compliant or potentially
+compromised state. For example, if it is determined that a firmware or software component no longer
+meets compliance requirements and the launch authentication mechanism fails, it may be possible
+to update that component and retry (recover) the launch. Recoverability generally requires that
+some component(s) of the TEE remain trusted, which can act as a “root” when other components are
+updated. (Further discussion on such recovery is provided later in section 7.)
+
+A hardware-based TEE uses hardware-backed techniques to provide increased security guarantees for
+the execution of code and protection of data within that environment. This assurance is often missing in
+approaches that do not use a hardware-based TEE.
+
+For more discussion of TEE definitions used in the industry, see the discussion at [6].
+
+### 4 Related Technologies
+
+The TAC conducted a survey of various terms in the industry related to protecting data in use, and
+composed the following Venn diagram of technologies:
+
+![Alt text](related-terminology.png "Related Terminology")
+
+Figure 1 - Venn diagram of various technologies used to protect data in use
+
+Unfortunately, unlike the term “confidential computing”, several of the terms used in the diagram have
+multiple competing definitions. For example, “privacy-preserving computation” is variously defined
+as being synonymous with multi-party computation, or covering both multi-party computation and
+homomorphic encryption, or even (e.g., in [2]) covering the entire space of protecting data in use.
+This diagram illustrates why we refer to confidential computing as protecting data in use by using a
+hardware-based TEE, to distinguish it from other techniques.
+
+**4.1 Security Comparisons**
+
+The following table compares a typical TEE implementation with typical implementations of two other
+emerging classes of solution that protect data in use, Homomorphic Encryption (HE) and Trusted Platform
+Modules (TPM).
```

Comment:
Homomorphic Encryption and Trusted Platform Modules should both be links, like they are in the whitepaper